src/create3-fastslam.cpp

```diff
@@ -21,6 +21,8 @@
 #include "lidar-subscriber.h"
 using namespace std::chrono_literals;
 
+#define VISUALIZE_ROB_PATH
```

Comment:
I figured out that it wouldn't work before because I had it as an #elifdef after #ifdef USE_ROB_ODOM. I will double check that its only defined in one place